src/Service.Tests/SqlTests/GraphQLCollationTests/MsSqlGraphQLCollationTests.cs

```diff
@@ -24,17 +28,29 @@ public static async Task SetupAsync(TestContext context)
 
         #region Tests
         /// <summary>
-        /// MsSql Capitalization Collation Tests
+        /// MsSql Collation Tests to ensure that GraphQL is working properly when there is a change in case sensitivity on the database
         /// </summary>
         [DataTestMethod]
-        [DataRow("books", "title", @"SELECT title FROM books ORDER BY title ASC FOR JSON PATH, INCLUDE_NULL_VALUES")]
-        [DataRow("books", "title", @"SELECT title FROM BOOKS ORDER BY TITLE ASC FOR JSON PATH, INCLUDE_NULL_VALUES")]
-        [DataRow("books", "title", @"SELECT title FROM Books ORDER BY Title ASC FOR JSON PATH, INCLUDE_NULL_VALUES")]
-        public async Task MsSqlCapitalizationResultQuery(string type, string item, string dbQuery)
+        [DataRow("comics", "title", @"SELECT title FROM comics ORDER BY title ASC FOR JSON PATH, INCLUDE_NULL_VALUES")]
+        [DataRow("authors", "name", @"SELECT name FROM authors ORDER BY name ASC FOR JSON PATH, INCLUDE_NULL_VALUES")]
+        [DataRow("fungi", "habitat", @"SELECT habitat FROM fungi ORDER BY habitat ASC FOR JSON PATH, INCLUDE_NULL_VALUES")]
+        public async Task MsSqlCaseSensitiveResultQuery(string type, string item, string dbQuery)
         {
-            await CapitalizationResultQuery(type, item, dbQuery);
+            string defaultCollationQuery = MsSqlCollationQuery(type, item, DEFAULT_COLLATION);
+            string newCollationQuery = MsSqlCollationQuery(type, item, CASE_SENSITIVE_COLLATION);
+            await CaseSensitiveResultQuery(type, item, dbQuery, defaultCollationQuery, newCollationQuery);
```

Comment:
nit: rename `CaseSensitiveResultQuery` to be a verb and better explain what the test is doing - e.g. TestQueryingWithCaseSensitiveCollation